README.md

```diff
@@ -94,6 +98,17 @@ const exampleButton = document.querySelector('[data-opens-menu]')
 const exampleMenuButton = new MenuButton(exampleButton)
 ```
 
+### `menuitemradio` example
```

Comment:
I would make this a bit more explicit. The fact that it uses `menuitemradio` is not really the first thing the user wants to know.